docs/README.md

```diff
@@ -150,6 +150,7 @@ Booster will fit like a glove in applications that are naturally event-driven li
 - **All the advantages of Serverless, without needing a degree in cloud technologies**: Serverless technologies are amazing and have made a project like Booster possible, but they're relatively new technologies, and while day after day new tools appear to make them easier, the learning curve is still quite steep. With Booster you'll take advantage of Serverless’ main selling points of high scalability and reduced hosting costs, without having to learn every detail from minute one.
 - **Event-sourcing by default**: Booster keeps all incremental data changes as events, indefinitely. This means that any previous state of the system can be recreated and replayed at any moment, enabling a whole world of possibilities for troubleshooting and auditing, syncing environments or performing tests and simulations.
 - **Booster makes it easy to build enterprise-grade applications**: Implementing an event-sourcing system from scratch is a challenging exercise that usually requires highly specialized experts. There are some technical challenges like eventual consistency, message ordering, and snapshot building. Booster takes care of all of that and more for you, lowering the curve for people that are starting and making expert lives easier.
+- **Choose your application infrastructure and don't get tied to a certain platform:** Booster provides a highly decoupled architecture that enables the possibility of integrating with ease new providers with different specifications, including a custom Multi-cloud provider, without affecting the framework specification.
```

Comment:
`Choose your application infrastructure and don't get tied to a certain platform` -> `Choose your application cloud and avoid vendor locking` This is actually the way it's usually said. The test is fine.
agreed?

Reply:
Awesome @rdoria1! I'm working on that. Thank you :D